src/current/v25.1/alter-job.md

```diff
@@ -0,0 +1,48 @@
+---
+title: ALTER JOB
+summary: Use the ALTER JOB statement to transfer ownership of a job between users or roles.
+toc: true
+---
+
+{% include_cached new-in.html version="v25.1" %} The `ALTER JOB` statement transfers ownership of a [job]({% link {{ page.version.version }}/show-jobs.md %}) between [users]({% link {{ page.version.version }}/create-user.md %}) or [roles]({% link {{ page.version.version }}/create-role.md %}).
+
+## Required privileges
+
+To alter job ownership, the user must be one of the following: 
+
+- The current job owner.
+- A member of the role that is the current owner. 
+- An `admin`.
```

Comment:
this first mention of the `admin` role could link to https://www.cockroachlabs.com/docs/v25.1/security-reference/authorization.html#admin-role